notebooks/sklearn-perceptron.ipynb

```diff
@@ -244,7 +244,7 @@
    "outputs": [],
    "source": [
     "n3 = np.concatenate((C3, y3), axis=1)\n",
-    "np.savetxt('./data/perp-c3.csv',n3,delimiter=',')"
+    "np.savetxt('../data/perp-c3.csv',n3,delimiter=',')"
```

Comment:
Was there a good reason to change the path of the datasets?

Reply:
Hi @ybouilla,
It is just that I keep data directory in root fedbiomed, and I forgot to revert it back before creating this PR.